MAINTAINERS.md

```diff
@@ -52,7 +52,15 @@ does not in itself necessitate a major release.
 
 ## Cutting a release
 
-TODO: https://github.com/target/lorri/issues/269
+To cut a new release:
+1. Look at the changes since the last release to [determine the next
+   version](#versioning-scheme) and increment the `version` field in
+   `Cargo.toml` accordingly.
```

Comment:
Maybe explicitly say something like: `consider if the changes since the last release change the API and [is a major version bump](...link to those docs...)`

Reply:
Thanks, I added something to that effect.